cosmos/chain/genesis.go

```diff
@@ -32,6 +30,65 @@ func NewGenesisKV(key string, value interface{}) GenesisKV {
 	}
 }
 
+func safeSet(data map[string]interface{}, value interface{}, path []interface{}) error {
```

Comment:
![image](https://github.com/user-attachments/assets/4e30c70c-3c95-46bc-a5df-c6645c54e72c)
this code looks like that piece